proxy/http/HttpTransact.cc

```diff
@@ -3460,7 +3460,7 @@ HttpTransact::HandleStatPage(State *s)
       s->hdr_info.client_response.value_set(MIME_FIELD_CONTENT_TYPE, MIME_LEN_CONTENT_TYPE, s->internal_msg_buffer_type, len);
     }
   } else {
-    s->hdr_info.client_response.value_set(MIME_FIELD_CONTENT_TYPE, MIME_LEN_CONTENT_TYPE, "text/plain", 9);
+    s->hdr_info.client_response.value_set(MIME_FIELD_CONTENT_TYPE, MIME_LEN_CONTENT_TYPE, "text/plain", 10);
```

Comment:
I completely agree a non-magic number approach would be good, I think we should probably make a ticket to get that check automated etc. then we can enforce it as a standard instead of finding them in reviews ;)